src/Page/Shared/View.elm

```diff
@@ -180,3 +181,24 @@ viewGuideTeaserList includeSummary teasers =
 
     else
         text ""
+
+
+embeddedVideoStyle : Style
+embeddedVideoStyle =
+    batch
+        [ height (pct 100)
+        , left zero
+        , position absolute
+        , top zero
+        , width (pct 100)
+        , maxWidth (px (Theme.Global.maxTabletLandscape / 3))
```

Comment:
This makes the video not full width on a bigger screen. So I thought, maybe do we want to center it? But leaving left until another decision is made.

Reply:
Sorry, I don't think I made that very clear - the styles I suggested were meant to entirely replace those inside embeddedVideoStyle, so that it would not have a maxWidth any more, it's no longer necessary with the new layout